### PR TITLE
Added linting and fixed files

### PR DESCRIPTION
**Pull request by JacksonZeitspace**
Note: I didn't do the test file yet because Catalin was in there and didn't want to create a bunch of conflicts

### CODE REVIEW

**Comment by brentlintner** [comment]
[Add a JS linter to the Stressmodel](https://trello.com/c/hrAG41zg/831-add-a-js-linter-to-the-stressmodel)

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "guard-for-in": 0,
+    "import/extensions": 0,
+    "no-await-in-loop": 0,
+    "no-plusplus": 0,
+    "no-prototype-builtins": 0,
+    "no-restricted-globals": 0,
+    "no-restricted-syntax": 0,
+    "object-shorthand": 0
+  }
+}
```
Comment (brentlintner) [inline .eslintrc.json:27]: Readme docs on how to lint?

File: db.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
-import 'dotenv/config';
-import pg from 'pg';
-import format from 'pg-format';
-import lodash from 'lodash';
-import readline from 'readline'
+import "dotenv/config";
+import pg from "pg";
+import format from "pg-format";
+import lodash from "lodash";
+import readline from "readline";
```
Comment (brentlintner) [inline db.js:5]: Dig the double quotes, personally. 🥳

File: main.js [hunk 3]
```diff
@@ -13,106 +14,250 @@ database will be updated with the corresponding level values.
...
-  {option: options.dbHost, definition: '-H --db-host [host name]', value: 'DB_HOST'},
-  {option: options.dbName, definition: '-N --db-name [database name]', value: 'DB_NAME'},
-  {option: options.waysTableName, definition: '-w --ways-table-name [table name]', value: 'WAYS_TABLE_NAME'},
-  {option: options.stressLevelColumnName, definition: '-s --stress-level-column-name [column name]', value: 'STRESS_LEVEL_COLUMN_NAME'},
-  {option: options.osmIdColumnName, definition: '-o --osm-id-column-name [column name]', value: 'OSM_ID_COLUMN_NAME'},
+const requiredDbArgs = [
+  {
+    option: options.dbUser,
+    definition: "-u --db-user [user name]",
+    value: "DB_USER",
```
Comment (brentlintner) [inline main.js:182]: Please can we not have trailing commas required lol. 🙏

File: package.json [hunk 4]
```diff
@@ -36,6 +36,13 @@
     "xml2object": "^0.1.2"
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/plugin-syntax-import-assertions": "^7.20.0",
+    "eslint": "^8.27.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "jshint": "^2.13.5",
```
Comment (brentlintner) [inline package.json:45]: Is JSHint required as well?

**Review by brentlintner — CHANGES_REQUESTED** [review]

**Comment by brentlintner** [comment]
Screen Shot 2022-11-15 at 15 35 36
 
<hr />

neurotic-sadge

**Comment by brentlintner** [comment]
LGTM in general, aside from a few nitpicks and opinionated pleadings. 😄

File: package.json (hunk 4 above)
Comment (JacksonZeitspace) [reply package.json:45]: oops, no. I used it at one point

File: main.js (hunk 3 above)
Comment (JacksonZeitspace) [reply main.js:182]: Yea I'll change it

File: .eslintrc.json (hunk 1 above)
Comment (JacksonZeitspace) [reply .eslintrc.json:27]: Added

**Review by brentlintner — APPROVED** [review]
danke

**Pull request merged**